@LinProp/LinProp.m

```diff
@@ -2,41 +2,58 @@
 % Michael Wollensack METAS - 06.07.2022
 % Dion Timmermann PTB - 22.06.2022
 %
-% LinProp Const:
-% a = LinProp(value)
+% This class supports the creation of uncertainty objects and subsequent
+% calculation with them as well as storage of the results. It can handle
+% complex-valued and multivariate quantities. Internally, LinProp objects
+% in MATLAB are wrappers of the .NET interface of the <a href="www.metas.ch/unclib">METAS UncLib library</a>.
 %
-% LinProp Input (RealUncNumber)
-% a = LinProp(value, standard_unc, (idof))
+% <strong>Commonly used Constructors</strong> (Round brackes indicate vectors)
+%   u = LinProp(value)
+%   u = LinProp(value, standard_unc, [description])
+%   u = LinProp(value, (covariance), [description])
+%  (u)= LinProp((value), (covariance), [description])
+%   u = LinProp((samples), 'samples', [description], [probability])
+%   u = LinProp(value, (sys_inputs), (sys_sensitivities), 'system')
+% See <a href="matlab:help LinProp.LinProp -displayBanner">List of all constructors</a>
 %
-% LinProp Input (RealUncNumber)
-% a = LinProp(value, standard_unc, description)
+% The values of u can be accessed through u<a href="matlab:help LinProp.Value -displayBanner">.Value</a> or <a href="matlab:help LinProp.get_value -displayBanner">get_value</a>(u), 
+% and the standard uncertainties through u<a href="matlab:help LinProp.StdUnc -displayBanner">.StdUnc</a> or <a href="matlab:help LinProp.get_stdunc -displayBanner">get_stdunc</a>(u).
+% Many common matlab functions are availibe, see <a href="matlab:methods(LinProp(1))">list of all methods.</a>
```

Comment:
matlab --> MATLAB

Reply:
Fixed in [cdc4c04](https://github.com/wollmich/metas-unclib-matlab-wrapper/pull/51/commits/cdc4c0448e6d5fe0f6e742d3456226742b2ff4a8).